mantidproject/mantid
# Add documentation for SESANS data handling algorithms

Docs pages were added for the new `LoadSESANS` and `SaveSESANS` algorithms. In addition, some property descriptions in the algorithm sources were improved, with some input from Adam Washington.

**To test:**

Build the docs, run the doc tests and make sure that everything looks right.

Fixes #20385 

**Release Notes** 

*Does not need to be in the release notes, as the addition of LoadSESANS and SaveSESANS are already mentioned*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
